utils/api/requests.js

```diff
@@ -3,9 +3,15 @@ import { fetcher } from './fetcher'
 
 export const getAllRequests = () => {
   const { data, error } = useSWR(`/quote_groups/mine.json`, fetcher)
+  // TODO: this hard codes a quote group ID for now, but it will need to dynamically receive the id from.. somewhere?
+  //const { sowData } = useSWR(`/quote_groups/512528/proposals.json`, fetcher)
```

Comment:
when using useSWR it returns data and error. (re: [docs](https://swr.vercel.app/docs/data-fetching)). you can't use a custom variable name here. the custom name would be defined below in the return statement.